### DISCUSSION TITLE
Terminal.write(string) not render full text with long text

### DISCUSSION THREAD

**Discussion by axetroy** [sourcelair/xterm.js#4321] [Q&A]
## Details
- Browser and browser version: Chrome 105
- OS version: windows
- xterm.js version: 5.0

### Steps to reproduce

```js
const term = new Terminal({
  allowProposedApi: true,
  convertEol: true,
  disableStdin: true
});

  term.loadAddon(unicode11Addon);
  term.loadAddon(fitAddon);

  term.unicode.activeVersion = "11";

  term.open(refTerminal.value!);
  fitAddon.fit();
  term.loadAddon(webCPU);

term.write('loooooooooooooooooooooooooooog text')
```

When the text to be rendered is very long, it only renders the second half, the first half is not rendered

The text bytes length is 118554

**Comment by jerch** [comment]
Plz provide more ctx info, from your current issue description all that can be said is - "there is something wrong". Also your repro code is not a real repro, it contains several undefined symbols.

**Comment by HL950328** [comment]
how do you resolve this question ?

**Reply by jerch** [reply]
The question cannot be resolved with given information. Normally the terminal has no issues to digest very long strings up to several MBs (still you should not do that if you care for screen refresh rate). The issue here is def. caused by something else.
